Source/Core/VideoCommon/AVIDump.cpp

```diff
@@ -76,8 +77,8 @@ bool AVIDump::Start(int w, int h)
 {
   s_pix_fmt = AV_PIX_FMT_RGBA;
 
-  s_width = w;
-  s_height = h;
+  s_width = g_Config.iDumpWidth ? g_Config.iDumpWidth : w;
```

Comment:
Right, lots of games output in all sorts of weird ratios. In this case, if you take 14 pixels off the top and bottom of SSBM (luckily, I've heard "14 pixels" is a magic number of sorts for emulating CRT vertical overscan), then it's a nice round 4:3 AR, and the menus and HUDs are all still well within the viewing area.